rubocop-airbnb/spec/rubocop/cop/airbnb/spec_constant_assignment_spec.rb

```diff
@@ -1,80 +1,64 @@
-describe RuboCop::Cop::Airbnb::SpecConstantAssignment do
-  subject(:cop) { described_class.new }
-
+describe RuboCop::Cop::Airbnb::SpecConstantAssignment, :config do
   it 'rejects constant definition inside of a describe block' do
-    source = [
-      'describe Someclass do',
-      '  CONSTANT = 5',
-      'end',
-    ].join("\n")
-
-    inspect_source(source)
-    expect(cop.offenses.size).to eq(1)
+    expect_offense(<<~RUBY)
+      describe Someclass do
+        CONSTANT = 5
+        ^^^^^^^^^^^^ Defining constants inside of specs can cause spurious behavior. [...]
+      end
+    RUBY
   end
 
   it 'allows constant defined inside of a module' do
-    source = [
-      'module Someclass',
-      '  CONSTANT = 5',
-      'end',
-    ].join("\n")
-
-    inspect_source(source)
-    expect(cop.offenses).to be_empty
+    expect_no_offenses(<<~RUBY)
+      module Someclass
+        CONSTANT = 5
+      end
+    RUBY
   end
 
-  it 'allows constant defined in global space' do
-    source = [
-      'CONSTANT = 5',
-    ].join("\n")
-
-    inspect_source(source)
-    expect(cop.offenses.size).to eq(1)
```

Comment:
FLAG: This test verifies that an error was raised, but the spec text says "allows".